plugins/BEdita/Core/src/Model/Entity/JsonApiAdminTrait.php

```diff
@@ -27,13 +27,23 @@
 
     /**
      * {@inheritDoc}
+     *
+     * @codeCoverageIgnore
+     */
+    protected function routeNamePrefix()
+    {
+        return 'api:admin';
+    }
+
+    /**
+     * {@inheritDoc}
      */
     protected function getLinks()
```

Comment:
Is this method still necessary?

Reply:
Giving answer to my own question: yes, we do.

Reply:
Yes, given the current routing structure. We could change it in a new PR/issue though